packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -473,10 +473,14 @@ class _AppBarState extends State<AppBar> {
       );
     }
 
-    return new Material(
-      color: widget.backgroundColor ?? themeData.primaryColor,
-      elevation: widget.elevation,
-      child: appBar,
+    return new Semantics(
```

Comment:
Can you add a test that verifies correct traversal order for app bar with a leading icon (e.g. hamburger menu), a title, and a trailing icon (e.g. three dots menu)? That's the traversal order we got most called out for in a11y reviews. Will be good to ensure with tests to keep it correct.

Reply:
Done.

Reply:
[direct link](https://github.com/flutter/flutter/pull/16253/commits/c19f8ed851aec31a826e205222625e7fe4fdbf47#diff-47a1cc33e1557fbf135f82b71ac0edeb)